src/directives/bind.js

```diff
@@ -27,6 +27,9 @@ export function handleAttributeBindingDirective(component, el, attrName, express
             } else {
                 el.checked = !! value
```

Comment:
Pretty sure instead of doing the "if typeof string" you could just stick the value in here

Reply:
I think if the value is e.g. `true` we don't want to set the `value` attribute to `"true"` which is not what we want. But I think the `el checked = !! value` also needs to be changed.

Reply:
Sorry my bad I misread value vs checked